libs/community/langchain_community/document_loaders/browserbase.py

```diff
@@ -0,0 +1,43 @@
+import os
+from typing import Iterator, List
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+
+
+class BrowserbaseLoader(BaseLoader):
+    """Create new Browserbase loader"""
+
+    def __init__(
+        self,
+        urls: List[str],
+        api_key: str = os.environ["BROWSERBASE_KEY"],
```

Comment:
See how other integrations pick up information from the enviornment. We generally do not want code accessing the environment during initialization time.

```
        values["api_key"] = get_from_dict_or_env(
            values, "api_key", "BROWSERBASE_API_KEY"
        )

```